tests/full_wlm/test_launch_orc_lsf_batch.py

```diff
@@ -0,0 +1,130 @@
+    """test single node orchestrator"""
+    exp_name = "test-launch-lsf-orc-batch"
+    exp = Experiment(exp_name, launcher="lsf")
+    test_dir = fileutils.make_test_dir(exp_name)
+
+    # batch = False to launch on existing allocation
+    orc = LSFOrchestrator(
+        6780, batch=True, project=wlmutils.get_test_account(), time="00:05", smts=1
+    )
+    orc.set_path(test_dir)
+
+    exp.start(orc, block=True)
+    status = exp.get_status(orc)
+
+    # don't use assert so that we don't leave an orphan process
+    if constants.STATUS_FAILED in status:
+        exp.stop(orc)
+        assert False
+
+    exp.stop(orc)
+    status = exp.get_status(orc)
+    assert all([stat == constants.STATUS_CANCELLED for stat in status])
+
+
+def test_launch_lsf_cluster_orc(fileutils, wlmutils):
+    """test clustered 3-node orchestrator"""
+    exp_name = "test-launch-lsf-cluster-orc-batch"
+    exp = Experiment(exp_name, launcher="lsf")
+    test_dir = fileutils.make_test_dir(exp_name)
+
+    # batch = False to launch on existing allocation
+    orc = LSFOrchestrator(
+        6780,
+        db_nodes=3,
+        batch=True,
+        project=wlmutils.get_test_account(),
+        time="00:03",
+        smts=1,
+    )
+    orc.set_path(test_dir)
+
+    exp.start(orc, block=True)
+    status = exp.get_status(orc)
+
+    # don't use assert so that orc we don't leave an orphan process
+    if constants.STATUS_FAILED in status:
+        exp.stop(orc)
+        assert False
+
+    exp.stop(orc)
+    status = exp.get_status(orc)
+    assert all([stat == constants.STATUS_CANCELLED for stat in status])
+
+
+def test_launch_lsf_cluster_orc_reconnect(fileutils, wlmutils):
+    """test reconnecting to clustered 3-node orchestrator"""
+
+    exp_name = "test-launch-lsf-cluster-orc-batch-reconect"
+    exp = Experiment(exp_name, launcher="lsf")
+    test_dir = fileutils.make_test_dir(exp_name)
+
+    # batch = False to launch on existing allocation
+    orc = LSFOrchestrator(
+        6780,
+        db_nodes=3,
+        batch=True,
+        project=wlmutils.get_test_account(),
+        time="00:05",
+        smts=1,
+    )
+    orc.set_path(test_dir)
+
+    exp.start(orc, block=True)
+
+    status = exp.get_status(orc)
+    # don't use assert so that orc we don't leave an orphan process
+    if constants.STATUS_FAILED in status:
+        exp.stop(orc)
+        assert False
+
+    exp.stop(orc)
+
+    exp_name = "test-orc-lsf-cluster-orc-batch-reconnect-2nd"
+    exp_2 = Experiment(exp_name, launcher="lsf")
+
+    checkpoint = osp.join(test_dir, "smartsim_db.dat")
+    reloaded_orc = exp_2.reconnect_orchestrator(checkpoint)
+
+    # let statuses update once
+    time.sleep(5)
+
+    statuses = exp_2.get_status(reloaded_orc)
+    for stat in statuses:
+        if stat == constants.STATUS_FAILED:
+            exp_2.stop(reloaded_orc)
+            assert False
+    exp_2.stop(reloaded_orc)
+
+
+def test_orc_converter_lsf_batch():
```

Comment:
Since this doesn't launch anything we should place it in the tests that can run with local. right?

Reply:
Done!